scripts/segmentation/train.py

```diff
@@ -192,6 +192,8 @@ def __init__(self, args, logger):
         # create criterion
         if 'icnet' in args.model:
             criterion = ICNetLoss(crop_size=args.crop_size)
+        elif 'danet' in args.model or 'danet' in args.model_zoo:
+            criterion=SegmentationMultiLosses()
```

Comment:
Add space before and after "="

Reply:
Thanks for your comments. I've made the changes based on them.